uvicorn/config.py

```diff
@@ -231,7 +231,7 @@ def __init__(
         debug: bool = False,
         reload: bool = False,
         reload_dirs: Optional[Union[List[str], str]] = None,
-        reload_delay: Optional[float] = None,
+        reload_delay: float = 0.25,
```

Comment:
I think None is valid here, it will be used in https://github.com/encode/uvicorn/blob/b1a45821a2ab916b4428bdcb9eba4a4a0cc8fb74/uvicorn/supervisors/basereload.py#L57 and https://docs.python.org/3/library/threading.html#threading.Event.wait can take None as a parameter

Reply:
Currently, `None` is converted to 0.25 on the `Config.__init__`. I think there's more reasoning for this value on the PR that introduced it...

Reply:
I suppose it is from b1a45821a2ab916b4428bdcb9eba4a4a0cc8fb74
I think it's better not to allow None to be passed as it would hang the app, wouldn't it ?

Reply:
No. It's been there even after this: https://github.com/encode/uvicorn/pull/774

Reply:
`0.25` was hard coded, and it allowed modification on #774